controllers/borrowingsController.js

```diff
@@ -0,0 +1,133 @@
+import Borrowing from "../models/borrowings.model.js";
+import Book from "../models/books.model.js";
+import Member from "../models/members.model.js";
+
+// borrow book
+export async function borrowBook(req, res) {
+    let member = await Member.findAll({where: {member_id: req.body.member_id}});
+    let book = await Book.findAll({where: {book_id: req.body.book_id}});
+    let borrowing;
+
+    try {
+        if (member && book.no_of_copies > 0) {
+            req.body.returned = false;
+            borrowing = await Borrowing.create(req.body);
+            // decrement number of copies of a book
+        }
+
+        if (borrowing) {
+            res.status(200).json({
+                success: true,
+                message: "Book borrowed successfully",
+                data: borrowing
+            });
+        } else {
+            res.status(500).json({
+                success: false,
+                message: "Borrowing book failed"
+            });
```

Comment:
Consider using a `400` `Bad request` for this; If it does not reach the catch statement, it is most likely an issue with the client request more than a server issue.